Fragments/Filter/LHE_BGenFilterEQ0_filter.dat

```diff
@@ -0,0 +1,6 @@
+LHEfilter = cms.EDFilter("LHEGenericFilter",
+    src = cms.InputTag("externalLHEProducer"),
+    NumRequired = cms.int32(0),
+    ParticleID = cms.vint32(5),
+    AcceptLogic = cms.string("EQ") # LT meaning < NumRequired, GT >, EQ =, NE !=
+)
```

Comment:
the file name could be misleading because it says "BGen"

Reply:
We need to define a convention for filter names. "Filter" is redundant. The whole folder should be cleaned up and fixed. Something for our clean-up session.